api/query_test.go

```diff
@@ -332,14 +333,16 @@ func TestFenceHappyHTTPResponse(t *testing.T) {
 			)
 		}
 
-		if len(parts[0]) != 0 {
-			msg := "Got %d bytes in metadata reply; want it to always be 0 in case '%s'"
-			t.Errorf(
-				msg,
-				len(parts[1]),
-				testcase.name,
-			)
+		metadata := string(parts[0])
+		expectedMetadata := `{
+			"shape": [` + strconv.Itoa(len(testcase.fence.Coordinates)) + `, 4]
```

Comment:
You really hate `fmt.Sprintf` 😄

Reply:
I think I can't use it in this case?
This is is a raw string literal (so that I can have all json quotes and so without problems and nice multiline).
I don't think it is possible for fmt to play nicely with it and it defeats purpose of raw string literal.